client/components/common/VersionString/index.js

```diff
@@ -41,11 +41,13 @@ const VersionString = ({
     linkRef,
     linkHref
 }) => {
+    const dateString = convertDateToString(date, 'YY.MM.DD');
+
     const body = (
-        <>
+        <span aria-label={`V${dateString}`}>
```

Comment:
In VoiceOver on MacOs, the non-link version of this markup reads with more complexity now. Here is what I experienced:

On update-database-impl:
On first focus, I hear: "Draft Review, V22.03.17, Column 5 of 7"
When I navigate to the next item, "Review Completed" immediately gets the focus.

On  this branch:
On first focus, I hear "V22.03.17, group, 1 of 2"
When I navigate to the next item, "V" is focused and read.
When I navigate to the next item, "22.03.17" is focused and read
When I navigate to the next item, the parent is focused again and "end of V22.03.17, group" is read.
When I navigate to the next item, "Review Completed" gets focus

Reply:
Note that the link version has identical output on both branches.

Reply:
Okay, so this is fascinating. In JSX, if you put `<b>{'V' + dateString}</b>` instead of `<b>V{dateString}</b>`, the issue where the screen reader reads the V separate from the date goes away.

If you do that the issue necessitating the aria-label goes away.

I had no idea until I started messing around with it.

Reply:
That's bizarre and kudos for finding that! We should definitely note this down somewhere. I tested it out and it sounds like expected!